shop/.dockerignore

```diff
@@ -9,3 +9,6 @@
 !composer.*
 !yarn.*
 !.stylelint*
+!tests/
+!setup_test
```

Comment:
I thought so too. But if you don't have **setup_test** and you try to run codecept, it fails saying that the file couldn't be found.